frontend/src/app/user/my-conferences/my-conferences.component.scss

```diff
@@ -0,0 +1,3 @@
+h1 {
+  font-family: "Lucida Sans Unicode", sans-serif;
```

Comment:
don't change fonts for one single page. The fonts on all pages should be the same